src/main/java/org/jetbrains/bsp/bazel/server/bsp/impl/BuildServerImpl.java

```diff
@@ -61,6 +61,12 @@ public void onBuildExit() {
     return buildServerService.workspaceBuildTargets();
   }
 
+  @Override
+  public CompletableFuture<Object> workspaceReload() {
+    // TODO
+    return null;
```

Comment:
It's probably best to return a [Completed Future](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html#completedFuture-U-) to make sure this does not break on the client side.